Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs

```diff
@@ -1,448 +1,589 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
-//
 using System;
 
-/// <summary>
-/// A conglimeration of open-source simplex libraries in C# with an emphasis on performance
-/// </summary>
-public class FastSimplexNoise
+namespace HoloToolkit
 {
-    private const double STRETCH_2D = -1.0 / 4.73205;
-    private const double STRETCH_3D = -1.0 / 6.0;
-    private const double STRETCH_4D = -1.0 / 7.23607;
-    private const double SQUISH_2D = 1.0 / 2.73205;
-    private const double SQUISH_3D = 1.0 / 3.0;
-    private const double SQUISH_4D = 1.0 / 7.23607;
-    private const double NORM_2D = 1.0 / 47.0;
-    private const double NORM_3D = 1.0 / 103.0;
-    private const double NORM_4D = 1.0 / 30.0;
-
-    private const Int64 SEEDVAL_1 = 6364136223846793005L;
-    private const Int64 SEEDVAL_2 = 1442695040888963407L;
-
-    private byte[] perm;
-    private byte[] perm2D;
-    private byte[] perm3D;
-    private byte[] perm4D;
-
-    private static double[] gradients2D = new double[]
+    /// <summary>
+    /// A conglomeration of open-source simplex libraries in C# with an emphasis on performance
+    /// </summary>
+    public class FastSimplexNoise
     {
-             5,  2,    2,  5,
-            -5,  2,   -2,  5,
-             5, -2,    2, -5,
-            -5, -2,   -2, -5,
-    };
+        private const double STRETCH_2D = -1.0 / 4.73205;
+        private const double STRETCH_3D = -1.0 / 6.0;
+        private const double STRETCH_4D = -1.0 / 7.23607;
+        private const double SQUISH_2D = 1.0 / 2.73205;
+        private const double SQUISH_3D = 1.0 / 3.0;
+        private const double SQUISH_4D = 1.0 / 7.23607;
+        private const double NORM_2D = 1.0 / 47.0;
+        private const double NORM_3D = 1.0 / 103.0;
+        private const double NORM_4D = 1.0 / 30.0;
+
+        private const Int64 SEEDVAL_1 = 6364136223846793005L;
+        private const Int64 SEEDVAL_2 = 1442695040888963407L;
+
+        private byte[] perm;
+        private byte[] perm2D;
+        private byte[] perm3D;
+        private byte[] perm4D;
+
+        private static double[] gradients2D = new double[]
+        {
+            5, 2, 2, 5,
+            -5, 2, -2, 5,
+            5, -2, 2, -5,
+            -5, -2, -2, -5,
+        };
 
-    private static double[] gradients3D =
-    {
-            -11,  4,  4,     -4,  11,  4,    -4,  4,  11,
-             11,  4,  4,      4,  11,  4,     4,  4,  11,
-            -11, -4,  4,     -4, -11,  4,    -4, -4,  11,
-             11, -4,  4,      4, -11,  4,     4, -4,  11,
-            -11,  4, -4,     -4,  11, -4,    -4,  4, -11,
-             11,  4, -4,      4,  11, -4,     4,  4, -11,
-            -11, -4, -4,     -4, -11, -4,    -4, -4, -11,
-             11, -4, -4,      4, -11, -4,     4, -4, -11,
+        private static double[] gradients3D =
+        {
+            -11, 4, 4, -4, 11, 4, -4, 4, 11,
```

Comment:
Hmm, the formatting looked nicer before this change, having each column align from top to bottom.